packages/saber-plugin-search/lib/index.js

```diff
@@ -79,18 +79,19 @@ exports.apply = (api, options) => {
   api.browserApi.add(join(__dirname, 'saber-browser.js'))
 
   if (api.dev) {
-    api.hooks.onCreatePages.tapPromise(ID, async () => {
-      db = await generateDatabase()
-    })
+    // api.hooks.onCreateRenderer.tapPromise(ID, async () => {
+    //   db = await generateDatabase()
+    // })
 
     api.hooks.onCreateServer.tap(ID, server => {
-      server.get('/_saber/plugin-search/:locale.json', (req, res) => {
-        const db = getLocale(req.params.locale)
-        if (db) {
+      server.get('/_saber/plugin-search/:locale.json', async (req, res) => {
+        db = await generateDatabase()
```

Comment:
maybe we can generate db when it's actually used, wdyt? 😄 @krmax44

Reply:
Yup, that seems to work!